import java.util.InputMismatchException;
import java.util.Scanner;


public class TestException {

	static int input(){
		int re = 0;
		Scanner scanner = new Scanner(System.in);
		
		re = scanner.nextInt();
		
		return re;
	}
	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try {
			System.out.print("Enter First number: ");
			int x = input();
			System.out.print("Enter Second number: ");
			int y = input();
			
			System.out.println(x/y);
		} catch (InputMismatchException e) {
			// TODO: handle exception
			System.out.println("Exception : div by zero");
		}catch (ArithmeticException e) {
			// TODO: handle exception
				System.out.println("Exception...");
		}
		
		System.out.print("Bye");
	}

}
